### ISSUE TITLE
Data looks incorrect, causing mypy to create false alarms

### ISSUE THREAD

**Issue by genodeftest** [pygobject/pygobject-stubs#6]
Take this valid peace of python code:

```
#!/usr/bin/python3

from gi.repository import GObject

foo = GObject.ParamFlags.READWRITE

print(foo)
```

With pygobject-stubs installed, mypy throws this error:

```
error: "Type[ParamFlags]" has no attribute "READWRITE"  [attr-defined]
    foo = GObject.ParamFlags.READWRITE
```
Which is expected when having a look at the code: The [GObject.pyi](https://github.com/pygobject/pygobject-stubs/blob/master/gi-stubs/repository/GObject.pyi) file does have a [`ParamFlags`](https://github.com/pygobject/pygobject-stubs/blob/master/gi-stubs/repository/GObject.pyi#L34) attribute, but that does not have a `READWRITE` attribute. Instead [`READWRITE` is an attribute to GObject](https://github.com/pygobject/pygobject-stubs/blob/master/gi-stubs/repository/GObject.pyi#L293). The same issue is true for each and everything in all the .pyi files. Is this by intention? Is it expected to be incompatible to mypy?

**Comment by genodeftest** [comment]
Or is this project officially dead together with https://github.com/pygobject/pgi?

**Comment by genodeftest** [comment]
Would you be interested in a rework based on [fakegir](https://github.com/strycore/fakegir)?

**Comment by lazka** [comment]
Hey, yes, in the current state this package isn't really useful.

There is work to improve things here:  https://github.com/pygobject/pgi-docgen/pull/176

**Comment by lovetox** [comment]
This should be fixed now since 055b9ef51444f7fa113736f0149f785868e1510e
